content/en/docs/concepts/data-collection.md

```diff
@@ -38,3 +38,15 @@ components as well as pipelines can be defined via YAML configuration.
 
 For more information about these components see the [configuration
 documentation](/docs/collector/configuration).
+
+## Repositories
+
+The OpenTelemetry project provides two versions of the Collector:
+
+- **[Core](https://github.com/open-telemetry/opentelemetry-collector/releases):**
```

Comment:
maybe worth mentioning it is functional for basic scenarios in case when the most common protocols are used and only basic customization is needed.

Reply:
modified language